client/landing/splash.tsx

```diff
@@ -125,10 +125,21 @@ const ButtonsContainer = styled.div`
   flex-direction: row;
   flex-wrap: wrap;
   justify-content: center;
+  width: 100%;
+`
+
+const ContainerButtonAligner = styled.div.attrs(p => p)<{ align: 'left' | 'right' | 'center' }>`
```

Comment:
I don't think you need the `.attrs(p => p)` here.

Reply:
Yeah that's just redundant. Haven't used `styled-components` for a while